src/modules/Builder/hooks/useAppletPrivateKeySetter.ts

```diff
@@ -1,8 +1,8 @@
 import { Encryption, getAppletEncryptionInfo, getParsedEncryptionFromServer } from 'shared/utils';
-import { useEncryptionCheckFromStorage } from 'shared/hooks';
+import { useEncryptionStorage } from 'shared/hooks/useEncryptionStorage';
```

Comment:
do you really need additional '/useEncryptionStorage'? if not, please remove in all places

Reply:
fixed